### ISSUE TITLE
APK download link doesn't exist

### ISSUE THREAD

**Issue by sokolovicandjela** [reoky/android-crackme-challenge#12]
Hi,
Could you please provide valid links for apk download? Currently there's just a file name and a hash value.

Thanx!

**Comment by Cvjark** [comment]
Same issue

**Comment by reoky** [comment]
Yeah, these challenges are out-of-date at this point friends. I have this repository mainly for archival reasons. ~
